client/src/components/Sub_RatingsReviews/Star.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export default function Star({
+  size = 50, percentage, id,
+}) {
```

Comment:
can you explain what id means in the input parameters?

Reply:
Individual IDs are passed to the star gradient